js/newSubmit.min.js

```diff
@@ -10,21 +10,21 @@ this.$getRow().find(".form-input-react").text()}},!1,"submit"!==f.mode));g.push(
-this.rem=a.remarks;this.refDefault=a.refDefault;this.srcDefault=a.srcDefault;this.prefSrc="edit"===f.mode?!1:"undefined"===typeof u?!1:u;this.locked="undefined"===typeof d?!1:d;this.isThisFirstEntry=this.constructor.isNextFirstEntry;this.constructor.isNextFirstEntry=!1}function y(b,a){return{appender:function(c){var l=this,e=$("<input/>");e.addClass("submit-textinput");this.locked&&e.prop("disabled",!0);h(e,b);if("function"===typeof a)e.on("input",function(b){b=a.call(l,e.val(),e,b);"string"===typeof b&&
-e.val(b)});e.appendTo(c)},getter:function(){return this.$getRow().find(".submit-textinput").val()},setter:function(b){var c=this.$getRow().find(".submit-textinput");c.val(b);"function"===typeof a&&a.call(this,b,c,void 0)},afterRemarks:!1}}function D(b){return{appender:function(a){var c=$("<input type='checkbox'/>");c.addClass("submit-checkinput");this.locked&&c.prop("disabled",!0);h(c,b);c.appendTo(a)},getter:function(){return this.$getRow().find(".submit-checkinput").prop("checked")},setter:function(a){this.$getRow().find(".submit-checkinput").prop("checked",
-a)},afterRemarks:!1}}function G(b){return{appender:function(a){var c=$("<textarea class='submit-hybrid-content'></textarea>");this.locked&&c.prop("disabled",!0);h(c,b);c.appendTo(a);c="https://github.com/"+f.repoInfo.full_name+"/tree/"+f.buildInfo.sha.substring(0,7)+"/";$("<div></div>").append("<label style='padding: 5px;'>Format:</label>").append($("<select class='submit-hybrid-format' style='display: inline-flex;'></select>").append($("<option value='txt'>Plain Text (You may indent with spaces)</option>")).append($("<option value='sm'></option>").text("Standard Markdown (rendered like README, links relative to "+
+g.push("Plugin association");g.push(new k("assocParent",f.fields.assocParent,"Associate Release of:","submit2-assoc-parent",N,function(){return{valid:!0}}));0<f.assocChildren.length&&g.push(new k("assocChildrenUpdates",f.fields.assocChildren,"Children Associate Releases","submit2-assoc-children-updates",H(f.assocChildren,!0),function(){return{valid:!0}}));g.push("Other details");g.push(new k("license",f.fields.license,"License","submit2-license",O,function(){return null!==this.getValue().custom&&
+200>this.getValue().custom.length?{valid:!1,message:"The custom license must be a open source license, and the full text is required."}:{valid:!0}}));g.push(new k("perms",f.fields.perms,"Permissions","submit2-perms",H(f.consts.perms,!0),function(){return{valid:!0}}));g.push(E=new k("authors",f.fields.authors,"Producers","submit2-authors",P,function(){return{valid:this.getValue().length>=E.type.minRows,message:"There must be at least 1 producer!"}}));t(F);setTimeout(q,500,E);var b=$(".form-table");
```

Comment:
When removing my code (back to original js only this is removed rest of changes persist.